tox.ini

```diff
@@ -92,46 +90,14 @@ source =
     .
 omit =
     .tox/*
-    */charmhelpers/*
-    src/templates/*
     unit_tests/*
     tests/*
+    src/templates/*
 
 [testenv:venv]
 basepython = python3
 commands = {posargs}
 
-[testenv:build]
-basepython = python3
-deps = -r{toxinidir}/build-requirements.txt
-commands =
-    charmcraft build
-
-[testenv:func-noop]
-basepython = python3
-commands =
-    functest-run-suite --help
-
-[testenv:func]
-basepython = python3
-commands =
-    functest-run-suite --keep-model
-
-[testenv:func-smoke]
-basepython = python3
-commands =
-    functest-run-suite --keep-model --smoke
-
-[testenv:func-dev]
-basepython = python3
-commands =
-    functest-run-suite --keep-model --dev
-
-[testenv:func-target]
-basepython = python3
-commands =
-    functest-run-suite --keep-model --bundle {posargs}
-
 [flake8]
-# Ignore E902 because the unit_tests directory is missing in the built charm.
-ignore = E402,E226,E902
+# E402 ignore necessary for path append before sys module import in actions
```

Comment:
Can we remove the E402 since we should no longer have this path appending stuff?

Reply:
Done